modules/compliance-scansetting-cr.adoc

```diff
@@ -12,5 +12,5 @@ To increase the default CPU and memory limits of the Compliance Operator, see _I
 
 [IMPORTANT]
 ====
-Increasing the memory limit for the Compliance Operator or the scanner pods is needed if the default limits are not sufficient and the Operator or scanner pods are ended by the Out Of Memory (OOM) process.
+Increasing the memory limit for the Compliance Operator or the scanner pods is needed if the default limits are not sufficient and the Operator or scanner pods are ended by the Out Of Memory (OOM) process. For more information, see (link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/security_and_compliance/compliance-operator#compliance-increasing-operator-limits_compliance-troubleshooting[Increasing Compliance Operator resource limits]). 
```

Comment:
For the link, maybe better to use https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/security_and_compliance/compliance-operator#compliance-increasing-operator-limits_compliance-troubleshooting